settings.gradle

```diff
@@ -62,10 +63,11 @@ include ':scripts:java:tools:dummytokens'
 
 project(':integrations:spark:spark-3.1:openhouse-spark-runtime').name = 'openhouse-spark-runtime_2.12'
 project(':apps:spark').name = 'openhouse-spark-apps_2.12'
+project(':apps:spark-3.5').name = 'openhouse-spark-apps-1.5_2.12'
```

Comment:
```suggestion
project(':apps:spark-3.5').name = 'openhouse-apps_spark-3.5_iceberg-1.5_2.12'
```

Reply:
@teamurko this is inconsistent with the existing spark3.1 naming convention on line 65. do you think we should still do it?

Reply:
i see that, i guess we'll have to change both artifacts, we can do that in a separate PR if needed